sentinel/tests/unit/schedule/reminders.spec.js

```diff
@@ -139,6 +140,9 @@ describe('reminders', () => {
       const reminder = { form: 'formA', some: 'config' };
       reminders.__set__('matchReminder', sinon.stub().resolves(date));
       reminders.__set__('sendReminders', sinon.stub().resolves());
+      sinon.stub(performance, 'now')
+        .onFirstCall().returns(0)
+        .onSecondCall().returns(1);
```

Comment:
I assume this was required due to a change in the behavior of `perf_hooks`?  Pretty wild that this was the only thing that broke!  :sweat_smile:

Reply:
Yes it seems like the `sinon.useFakeTimers()` call now doesn't work for performance.now because node significantly changed their implementation. This only affects performance.now and only node processes which explains why we only hit it in one place.